app/admin/dashboard.rb

```diff
@@ -0,0 +1,41 @@
+ActiveAdmin.register_page 'Dashboard' do
+
+  menu priority: 1, label: proc{ I18n.t('active_admin.dashboard') }
+
+  content title: proc{ I18n.t('active_admin.dashboard') } do
```

Comment:
Space missing to the left of {.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.